### ISSUE TITLE
Add "validation of variables" to the advanced features section

### ISSUE THREAD

**Issue by mattbrictson** [capistrano/capistrano.github.io#151]
I removed this from the README in https://github.com/capistrano/capistrano/pull/1559; this needs to be added to the website instead.

`````` diff
-## Validation of variables
-
-To validate a variable, each time before it is set, define a validation:
-
-```ruby
-validate :some_key do |key, value|
-  if value.length < 5
-    raise Capistrano::ValidationError, "Length of #{key} is too short!"
-  end
-end
-```
-
-Multiple validation can be assigned to a single key. Validations will be executed
-in the order of registration.
``````

**Comment by will-in-wi** [comment]
PR for this in https://github.com/capistrano/documentation/pull/156

**Comment by mattbrictson** [comment]
I think PR #156 is incomplete. We still need a link in the sidebar to get to the validation page, and I believe the new validation page is not being processed by Jekyll because it lacks the title/layout front-matter.

**Comment by will-in-wi** [comment]
Good catch. I'll throw in a supplemental PR.

**Comment by will-in-wi** [comment]
Added PR.

**Comment by mattbrictson** [comment]
Fixed by #158.
